docs/source/_templates/layout.html

```diff
@@ -13,4 +16,8 @@
   ga('send', 'pageview');
 
 </script>
+<script data-goatcounter="https://docs-conda-io.goatcounter.com/count"
+        async src="//gc.zgo.at/count.js"></script>
```

Comment:
Just like https://github.com/conda/conda-build/pull/5093#discussion_r1413564562, we should host `count.js` ourselves